index.js

```diff
@@ -244,106 +242,85 @@ const getSettings = function(options) {
  *         they need to be exported to rollup as an Array.
  */
 const generateRollupConfig = function(options) {
-  options = options || {};
   const settings = getSettings(options);
 
   /* make a build with the specifed settings */
   const makeBuild = (buildType, buildOverrides, buildSettings = settings) => {
-    const b = Object.assign({}, {
-      plugins: buildSettings.plugins[buildType],
+    const b = Object.assign({
+      // never clear screen during watch
+      watch: {clearScreen: false},
+      // map plugin names from strings to primed plugins
+      // but only if the plugin is a string and not a
+      // primed plugin already.
+      plugins: buildSettings.plugins[buildType].map((p) => typeof p !== 'string' ? p : buildSettings.primedPlugins[p]),
       external: buildSettings.externals[buildType],
-      input: buildSettings.input
+      input: buildType === 'test' ? buildSettings.testInput : buildSettings.input
     }, buildOverrides);
 
-    // map plugin names from strings to primed plugins
-    // but only if the plugin is a string and not a
-    // primed plugin already.
-    b.plugins = b.plugins.map((pluginName) => {
-      if (typeof pluginName !== 'string') {
-        return pluginName;
-      }
-
-      return buildSettings.primedPlugins[pluginName];
-    });
-
-    const changeOutput = (o) => {
-      o.banner = buildSettings.banner;
-      o.globals = buildSettings.globals[buildType];
+    const changeOutput = (o) => Object.assign({
+      banner: buildSettings.banner,
+      globals: buildSettings.globals[buildType]
+    }, o);
 
-      return o;
-    };
-
-    if (!Array.isArray(b.output)) {
-      b.output = changeOutput(b.output);
-    } else {
-      b.output = b.output.map(changeOutput);
-    }
+    // keep output a non-array
+    b.output = (!Array.isArray(b.output) ? changeOutput(b.output) : b.output.map(changeOutput));
 
     return b;
   };
 
   /* all rollup builds by name. note only object values will be used */
   const builds = {
+    test: makeBuild('test', {
+      output: [{
+        name: `${settings.exportName}Tests`,
+        file: 'test/dist/bundle.js',
+        format: 'iife'
+      }]
+    }),
     browser: makeBuild('browser', {
       output: [{
         name: settings.exportName,
         file: `dist/${settings.distName}.js`,
         format: 'umd'
+      }, {
+        name: settings.exportName,
```

Comment:
Merged the two browser builds and the two module builds. This saves us about ~6s in VHS when we build everything.